resource/js/vocab-search.js

```diff
@@ -225,12 +225,26 @@ const vocabSearch = Vue.createApp({
   template: `
     <div class="d-flex my-auto ms-auto">
       <div class="d-flex justify-content-end input-group ms-auto" id="search-wrapper">
-        <select class="btn btn-outline-secondary dropdown-toggle" data-bs-toggle="dropdown-item"
-          v-model="selectedLanguage"
-          @change="changeContentLangAndReload($event.target.value)"
-          aria-label="Select search language">
-          <option class="dropdown-item" v-for="(value, key) in languageStrings" :value="key">{{ value }}</option>
-        </select>
+
+      <div class="dropdown" id="language-selector">
+        <a class="btn btn-outline-secondary dropdown-toggle"
```

Comment:
Should this be a button like here? https://getbootstrap.com/docs/5.2/components/dropdowns/#single-button

Using a button seems to at least fix the size of the dropdown toggle

Reply:
Button is fine too. It can be done with either - the next example on the linked document is with <a> -elements. I'll have a look at how the choise of the element affects accessibiliy, mobile user agents etc.